lxd/instances_get.go

```diff
@@ -304,12 +280,12 @@ func doInstancesGet(s *state.State, r *http.Request) (any, error) {
 		return nil
 	})
 	if err != nil {
-		return nil, err
+		return response.SmartError(err)
 	}
 
 	userHasPermission, err := s.Authorizer.GetPermissionChecker(r.Context(), auth.EntitlementCanView, entity.TypeInstance)
 	if err != nil {
-		return nil, err
+		return response.InternalError(err)
```

Comment:
Lets not wrap this in response.InternalError() and instead use response.SmartError() as we dont always know that GetPermissionChecker want us to return an internal error.

Reply:
Done!